addb2/dump.c

```diff
@@ -93,6 +93,7 @@
 #include "scripts/systemtap/kem/kem_id.h"
 #include "addb2/addb2_internal.h"
 #include "lib/trace.h"
+#include "lib/thread.h"
```

Comment:
Please specify function that is used in comments:
Example: 
`#include "lib/thread.h"       /* m0_pid() */  `